Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -451,41 +451,68 @@ void doPublishRawImu(void) {
 /*
  * Publish bumper data to ros.
  */
+static const int NUM_BUMPER_SAMPLES(7);
+
 void doPublishBumpers(void) {
-  // TODO(mwomack): Get readings from robot
-  static const int MIN_RAW_LEFT_BUMPER(650);
-  static const int MAX_RAW_LEFT_BUMPER(450);
-  static const double RANGE_RAW_LEFT_BUMPER(MAX_RAW_LEFT_BUMPER - MIN_RAW_LEFT_BUMPER);
-  static const double MAX_FORCE_LEFT_BUMPER(1.0);
-  static const int MIN_RAW_RIGHT_BUMPER(650);
-  static const int MAX_RAW_RIGHT_BUMPER(450);
-  static const double RANGE_RAW_RIGHT_BUMPER(MAX_RAW_RIGHT_BUMPER - MIN_RAW_RIGHT_BUMPER);
-  static const double MAX_FORCE_RIGHT_BUMPER(1.0);
+  // These values come from samples taken directly from the robot.
+  // Distances are in meters.
+  static const int BUMPER_BASE_LEFT(620);
+  static const int BUMPER_RAW_SAMPLES_LEFT[NUM_BUMPER_SAMPLES] = { 0, 60, 144, 216, 250, 277, 287 };
```

Comment:
So, in general, I hate things like this and feel that they should be read from a file or whatever, but that can't be done here. In the future (or now), let's make this a rosparam read in from a yaml if rosserial allows it.
At a minimum, I feel like this needs many more comments to explain what the data is and how it is used.